monai/apps/generation/maisi/networks/autoencoderkl_maisi.py

```diff
@@ -0,0 +1,891 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import gc
+from typing import Sequence
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from generative.networks.nets.autoencoderkl import AttentionBlock, AutoencoderKL, ResBlock
```

Comment:
```suggestion
from monai.utils import optional_import
autoencoderkl, has_generative = optional_import("generative.networks.nets", name="autoencoderkl")
```

Reply:
Hi @KumoLiu, I have updated the script. Shall we resolve this conversation?